ecommerce/products/templates/home.html

```diff
@@ -5,13 +5,19 @@ <h2>Recently added</h2>
 <ul>
     <li>
         <div>
+            <a href="{{ product.id }}">
             <div><strong>Name: </strong>{{ product.name }}</div>
+            </a>
+            <a href="{% url 'product-by-type' product.product_type %}">
             <div><strong>Type: </strong>{{ product.product_type }}</div>
+            </a>
```

Comment:
It would be good to add some indentation here:
```
            <a href="{% url 'product-by-type' product.product_type %}">
                <div><strong>Type: </strong>{{ product.product_type }}</div>
            </a>
```